samza-core/src/main/scala/org/apache/samza/util/Util.scala

```diff
@@ -148,17 +148,28 @@ object Util extends Logging {
    * @return re-written config
    */
   def rewriteConfig(config: Config): Config = {
-    def rewrite(c: Config, rewriterName: String): Config = {
-      val rewriterClassName = config
-              .getConfigRewriterClass(rewriterName)
-              .getOrElse(throw new SamzaException("Unable to find class config for config rewriter %s." format rewriterName))
-      val rewriter = Util.getObj(rewriterClassName, classOf[ConfigRewriter])
-      info("Re-writing config with " + rewriter)
-      rewriter.rewrite(rewriterName, c)
+    config.getConfigRewriters match {
+      case Some(rewriters) => rewriters.split(",").foldLeft(config)(rewrite(_, _))
+      case _ => config
     }
+  }
 
+  private def rewrite(config: Config, rewriterName: String): Config = {
+    val rewriterClassName = config
+      .getConfigRewriterClass(rewriterName)
+      .getOrElse(throw new SamzaException("Unable to find class config for config rewriter %s." format rewriterName))
+    val rewriter = Util.getObj(rewriterClassName, classOf[ConfigRewriter])
+    info("Re-writing config with " + rewriter)
+    rewriter.rewrite(rewriterName, config)
+  }
+
+  def runRegexTopicGenerator(config: Config): Config = {
```

Comment:
call this "invokeRegexTopicRewriter"?

Reply:
Does this need to be a public util method? Just inline it in the JobModelManager? Would prefer to not make Util a dumping ground for helper methods again :)